app/i18n/ja/people.ts

```diff
@@ -207,24 +228,28 @@ const people: People = {
           'ワシントン大学電気・コンピュータ工学教授',
           'スタンフォード大学電気工学博士',
         ],
+        link: 'https://people.ece.uw.edu/atlas/',
       },
       {
         img: NobuyoOzawa,
         name: '小澤信与',
         role: '日本臨床開発リード',
         texts: ['臨床開発部グループマネージャー'],
+        link: 'https://virufy.org/en/our-org',
```

Comment:
link to this page will break once 6.0 becomes virufy.org, as there isn't an `/our-org` route. I reached out to Moeka from Japan team to see if she has a link to Nobuyoshi's profile.

Reply:
pending

Reply:
Still no response from Moeka. I say just push what we have now and add the link later, if she has one. I don't think that Ozawa has a link.

Reply:
She provided a link. I added it to recent commit.